README.md

```diff
@@ -36,6 +36,18 @@ for(auto setting : settings.toStdMap()) {
 }
 ```
 
+## How to use EditorConfig Qt5 Core (Cmake)
+
+The easiest way, should be to use CPM (https://github.com/cpm-cmake/CPM.cmake). Then,
+inside your application do:
+
+``` CMake
+# Or any other sha1 you want.
+CPMAddPackage("gh:editorconfig/editorconfig-core-qt#master")
+```
+
+The follow instructions for QMake for code integration.
+
 ## Development
 
 This has been developed with Qt 5.14, however previous versions *may* work. This has only been tested on Windows.
```

Comment:
TODO: should this be removed? We know this works on Qt6, and Linux+OSX

Reply:
Once we get it working then yeah this will probably need removed or clarification added as to what it supports.

Reply:
not handling this right now, ok.